app/2.0/docs/tools/glossary.md

```diff
@@ -0,0 +1,162 @@
+
+If you run `polymer install --variants`, the `polymer install` tool creates separate builds for your app or element using the versions of Polymer you specify. You can use the [Polymer development server](#polyserve) to test these variants. 
+
+## dynamic dependency
+A dynamic dependency is a dependency that is imported at runtime, using a mechanism such as `importHref`. Compare [static dependency](#static-dependency).
+
+# E
+## eager loading
+Eager loading refers to the process of loading dependencies statically, rather than on demand. Compare [lazy loading](#lazy-loading).
+
+## entry point
+The HTML actually loaded by the browser to start the application. Usually `index.html`. In an application built with the [app shell](#app-shell) architecture, the entrypoint is loaded for many different URLs.
+
+# F
+## fragment
+A lazily-loaded HTML file. Fragments are used to define bundles and are often associated with routes or non-critical resources.
+
+# G
+## git
+A distributed version control system. bower depends on git to install components from their GitHub repositories.
+
+## grunt
+A JavaScript task automation tool that can be used to set up a custom build process using the [`polymer-build` library](#polymer-build).
+
+## gulp
+A streamed JavaScript task runner that can be used to set up a custom build process using the [`polymer-build` library](#polymer-build).
+
+# H
+## hydrolysis
+This tool has been renamed to [Polymer Analyzer](#polymer-analyzer). It is a static analysis framework for Web Components. 
+
+# L
+## lazy loading
+Lazy loading takes advantage of routing to load elements only when they are required for render. Used correctly, lazy loading can improve performance.
+
+For an example implementation of lazily-loaded elements, see the [Shop app](https://github.com/Polymer/shop). In the Shop app, views are elements which are instantiated only when they are routed to. 
+
+## linter
+A linter is a piece of software that analyzes code for suspicious-looking constructions that can potentially cause errors; for example, division by zero or data that will go out of bounds. The Polymer tools include [`polylint`](#polylint), which does this for Polymer apps and elements.
+
+# M
+## monolithic
+A monolithic app is one that has been implemented without a modular design pattern. 
+
+# N
+## Node.js
+An event-driven JavaScript runtime environment. The [Polymer CLI](#polymer-cli) and [bower](#bower) are both built on Node.
+
+## npm
+Package management system for [Node.js](#node-js).
+
+# P
+## package.json
+A metadata file used by [npm](#npm) to manage project dependencies.
+
+## polymer-build
+`polymer-build` is a library for building Polymer projects. `polymer-build` allows you to completely customize your build and combine additional streams and build tasks in any order.
```

Comment:
- we get a lot of questions about polymer-build vs the CLI build command. A quick sentence about how they're different here would be helpful.